src/main/java/com/booking/app/controller/RoomController.java

```diff
@@ -0,0 +1,25 @@
+package com.booking.app.controller;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+@RestController
+public class RoomController extends BaseController {
+    private static final String API_NAME = "rooms";
+
+    @PostMapping(RoomController.API_NAME + "/{id}")
+    public ResponseEntity<String> createRoom(@RequestBody long id) {
+        return ResponseEntity.ok("Room created!");
+    }
+
+    @PutMapping(RoomController.API_NAME + "/{id}")
+    public ResponseEntity<String> updateRoomStatus(@PathVariable long id) {
```

Comment:
Again as last comment. You are only sending an ID. How do know what fields to change?